config.lua

```diff
@@ -2,6 +2,7 @@ Config = Config or {}
 Config.AutoRespawn = false --True == auto respawn cars that are outside into your garage on script restart, false == does not put them into your garage and players have to go to the impound
 Config.SharedGarages = false   --True == Gang and job garages are shared, false == Gang and Job garages are personal
 Config.VisuallyDamageCars = true --True == Visually damage cars that go out of the garage depending of body damage, false == Do not visually damage cars (damage is still applied to car values)
+Config.SharedPublicGarages = true --Ture All public garages can access all vehicle of player that parked in garage type "public", -- False player can't access another public garages from anywhere (this is original behavior of script)
```

Comment:
Please set this to `false` by default.
Thanks!

Reply:
Are you sure? The reason for setting this to true by default is to avoid breaking changes for other users who rely on the old code. because them use code that have shared public garages from the least beginning.

Reply:
I am not familiar with how this originally functioned.
Garages have not been shared for more than a year. At this point it is being treated as a new optional feature, as the large majority of servers are not using the original qb-garages code.

After getting the opinions of the other devs, we agreed it should be set to false by default, as this has not been the default behavior for quite some time.